extensions/multipass/src/index.tsx

```diff
@@ -41,36 +40,61 @@ interface MultipassImage {
 
 export default function Command() {
   const [images, setImages] = useState<Array<MultipassImage>>([]);
+  const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     let running = true;
     async function fetchImages() {
       while (running) {
-        const { list } = await mp.localImages();
-        setImages(list);
+        try {
+          const { list } = await mp.localImages();
+          setImages(list);
+        } catch (e) {
+          await showToast(Toast.Style.Failure, "Error", "Failed to fetch images");
+          running = false;
+        } finally {
+          setIsLoading(false);
+        }
       }
     }
     fetchImages();
     return () => {
       running = false;
+      setIsLoading(false);
     };
   }, []);
 
   return (
-    <List>
+    <List isLoading={isLoading}>
       {images.map((image) => (
         <List.Item
           key={image.name}
           title={`${iconForState(image.state)}  ${image.name}`}
           actions={
             <ActionPanel>
-              <Action
-                title="Start or stop"
-                onAction={() => {
-                  primaryAction(image.name, image.state);
-                }}
-              />
-              <Action title="Suspend" onAction={() => secondaryAction(image.name, image.state)} />
+              {["Suspended", "Stopped"] && (
```

Comment:
Looks like something that is always true

Reply:
It does, thanks! Can you check it again? Feel free to modify for further yourself 🙂